docs/AboutUs.md

```diff
@@ -5,55 +5,67 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+This project is part of the NUS module CS2103 Software Engineering. 
+Our lecturer is Professor Damith Chatura Rajapakse.
+You can reach him at the email `damithch@comp.nus.edu.sg`.
 
 ## Project team
 
-### John Doe
+### Damith Chatura Rajapakse
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/damithc.png" width="200px">
 
 [[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/damithc)]
 
 * Role: Project Advisor
 
-### Jane Doe
+### Jaya Rengam
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jayarengam.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/jayarengam)]
+[[portfolio](team/jayarengam.md)]
 
 * Role: Team Lead
-* Responsibilities: UI
+* Responsibilities: Deliverables and deadlines
 
-### Johnny Doe
+### Goh Siau Chiak
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/sc-arecrow.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/sc-arecrow)]
+[[portfolio](team/sc-arecrow.md)]
 
 * Role: Developer
-* Responsibilities: Data
+* Responsibilities: Testing
 
-### Jean Doe
+### Lim Jin Feng
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jflim98.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/jflim98)]
+[[portfolio](team/jflim98.md)]
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: UI
 
-### James Doe
+### Joshua Chew Jian Xiang
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/josuaaah.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/josuaaah)]
+[[portfolio](team/josuaaah.md)]
 
 * Role: Developer
-* Responsibilities: UI
+* Responsibilities: Code Quality
+
+### Theodore Leebrant
+
+<img src="images/theodoreleebrant.png" width="200px">
```

Comment:
This image cannot be used, should be replaced with an image of the person or a placeholder image.

Reply:
Have replaced image with a placeholder image.